common/workunit/workunit.cpp

```diff
@@ -11216,6 +11217,16 @@ __int64 CLocalWUResult::getResultTotalRowCount() const
     return p->getPropInt64("totalRowCount", -1);
 }
 
+StringBuffer &CLocalWUResult::getResultTotalRowCountString(StringBuffer &str) const
+{
+    __int64 count = getResultTotalRowCount();
+    if (count == -1)
+        str.append("[unknown rowcount]");
```

Comment:
V. Minor:  I don't think `rowcount` is an actual word?  How about `[??? rows]`.
When the user is viewing these it might be easier to scan:
![image](https://github.com/hpcc-systems/HPCC-Platform/assets/966863/4fbf91f2-f248-42ba-990a-43eb4955b415)